### DISCUSSION TITLE
Review : Earnapp

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#7] [Announcements]
Here is my current dashboard following 1 month of use of earnapp application :

![image](https://user-images.githubusercontent.com/26048157/151464797-1b5e5ba0-2732-4352-8601-04ef9a939f6a.png)

Some info on the devices : I actually run 4 devices (The $0 revenues one are device that changed their device id when i rebuilded docker image). 3 devices are running on my home connection, 1 device is running on my 4G connection. I notice I have similar traffic on both connection (my 3 devices on home are using same bandwitdh as my 1 device on 4G, confirming there is no noticeable gain if you multiply instances on same IP, but there will be gain if you have several IPs). 

This passive income app is by far the most profitable one for me (and a must have).
Revenues are regular and around $20 / month.
Payout were done ($5 min) using paypal : I confirm I received the money (2 payout done manually and then I setted automatic payout and received another one). No unexpected fees on the transaction.

A must have : https://earnapp.com/i/p34wpf4 (with my account as referral) : Don't forget if you use my docker stack to log on portainer http gui (port 9000), then exec command shell on earnapp container (/bin/sh  not bin/bash) and type "earnapp showid" to get the device id that you must add in your dashboard.  

Don't hesitate to post any additionnal information on this application.

**Comment by lyenliang** [comment]
My Earnapp income is much less than yours. :-(
<img width="1145" alt="Screen Shot 2022-07-20 at 5 25 13 PM" src="https://user-images.githubusercontent.com/3468329/179947984-2e3bc5a5-b921-4631-b6bb-29b379bb73b6.png">
